tests/e2e/fixtures/storage_efs_dependencies.py

```diff
@@ -358,11 +374,14 @@ def on_delete():
                 MountTargetId=mount_target_id,
             )
 
+        wait_on_mount_target_status("deleted", efs_client, fs_id)
+
         # Delete the Filesystem
+
         efs_client.delete_file_system(
             FileSystemId=fs_id,
         )
-        wait_on_efs_status("deleted", efs_client, fs_id)
+        wait_on_efs_status("deleting", efs_client, fs_id)
```

Comment:
why not "deleted" anymore?

Reply:
Actually the dynamic provisioning here is still missing security group deletion, let me fix that.

Reply:
done.